infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultClusterManager.java

```diff
@@ -199,7 +199,7 @@ public static TopologyDescriptor readConfig() {
         for (int i = 0; i < activeNodeNames.size(); i++) {
             log.info("Active Cluster Name {}, IpAddress {}", activeNodeNames.get(i), activeNodeHosts.get(i));
             NodeDescriptor nodeInfo = new NodeDescriptor(activeNodeHosts.get(i),
-                    activeLogReplicationPort, ACTIVE_CLUSTER_NAME, UUID.fromString(activeNodeIds.get(i)), UUID.fromString(standbyNodeIds.get(i)));
+                    activeLogReplicationPort, ACTIVE_CLUSTER_NAME, UUID.fromString(activeNodeIds.get(i)), UUID.fromString(activeNodeIds.get(i)));
```

Comment:
shouldn't the connectionId be the IP?

Reply:
Yes, in the test case, the connectionId should be the IP but it is not used.
I am thinking if I can change the type of connectionId and nodeId from UUID to String. What do you think?

Reply:
looks like you are getting the id of active nodes 2 times.  Shouldnt the 2nd one be standbyNodeIds?

Reply:
Yes, I was thinking that too... the adapter should do the validation if it only accepts a UUID... I would say changing both to String is fine...

Reply:
@pankti-m the first id is connection id, and the second one is node id.